extension/view/static/dataConsent.html

```diff
@@ -0,0 +1,110 @@
+<html>
+<body>
+<style>
+
+ /* The switch - the box around the slider */
+ .switch {
+  position: relative;
+  display: inline-block;
+  width: 60px;
+  height: 34px;
+}
+
+/* Hide default HTML checkbox */
+.switch input {
+  opacity: 0;
+  width: 0;
+  height: 0;
+}
+
+/* The slider */
+.slider {
+  position: absolute;
+  cursor: pointer;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background-color: #ccc;
+  -webkit-transition: .4s;
```

Comment:
Do you target such old browsers that a -webkit- variant is needed? According to https://caniuse.com/?search=transition it should be fairly safe to only use the variant without vendor prefix.